11/Task11/Class1.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Task11
+{
+    public class Tour
+    {
+        public string Name { get; }
+        public readonly int Code;
+        public int Dur;

```

Comment:
В задании (сноска) рекомендовалось использовать тип `TimeSpan`. Вы используете `int`. Если тип `TimeSpan` однозначно определяет временной интервал, то в вашем случае при использовании типа `int` выбрано крайне неудачное имя для поля, ведь не понятно, в каких единицах измерять продолжительность тура (в часах, днях, неделях?). То есть нужно было либо так
```suggestion
        public TimeSpan Dur;
```
либо, например, так
```suggestion
        public int DurationInDays;
```